apps/server/test/modules/authentication/account.e2e-spec.ts

```diff
@@ -3,7 +3,7 @@ import { ExecutionContext, INestApplication } from '@nestjs/common';
 import request from 'supertest';
 import { Request } from 'express';
 import { EntityManager } from '@mikro-orm/core';
-import { ServerTestModule } from '@src/server.module';
+import { ServerTestModule } from '@src/modules/server/server.module';
```

Comment:
Similar '@src/modules/server' (i do not put more commits to the PR with same note) but interessting point the testing modul of the server is needed in multiple other modules for the api tests.